src/app/space/create/deployments/deployments-donut/deployments-donut.component.ts

```diff
@@ -54,7 +54,7 @@ export class DeploymentsDonutComponent implements OnInit {
   }
 
   getDesiredReplicas(): number {
-    if (this.desiredReplicas) {
+    if (this.desiredReplicas !== undefined) {
```

Comment:
I think this leaves getDesiredReplicas itself returning `undefined` if `this.desiredReplicas` is `undefined`. Maybe this should return 0, or some error-case sentinel (-1) in this case? This is another case of something that will probably change once this component is getting data from a service and using Observables, though.

Reply:
The full function definitions is:

````
 +    if (this.desiredReplicas !== undefined) {
 +      return this.desiredReplicas;
 +    }
 +
 +    // TODO: acquire replicas from service
 +
 +    return 1;
```

Doesn't this do what you mean, or did I miss something?

Reply:
Ah right, sorry. Nevermind that then :)